src/client/components/pages/parts/editor-profile.js

```diff
@@ -39,9 +39,14 @@ class EditorProfileTab extends React.Component {
 
 	renderBasicInfo() {
 		const {user, editor} = this.props;
-		const {cachedMetabrainzName, metabrainzUserId} = editor;
+		const {cachedMetabrainzName,
+			metabrainzUserId,
+			name,
+			gender,
+			birthDate} = editor;
 		const createdAtDate = (new Date(editor.createdAt)).toUTCString();
 		const lastActiveDate = (new Date(editor.activeAt)).toUTCString();
+		const birthday = (new Date(birthDate)).toUTCString();
```

Comment:
We should format this as YYYY-MM-DD - but since date/time display needs an overhaul across the site, you don't have to worry about this. If you want to anyway, please add Moment.js as a dependency then require that and use it to format.